### DISCUSSION TITLE
Default easing curves

### DISCUSSION THREAD

**Discussion by Storm840** [keiwando/toonsquid#610] [Ideas]
So it would be cool to be able to change the default easing curve (for now the default easing curve is "linear") so basically when you change that default E.C (Easing Curve), every time you make a new keyframe it would put it automatically to this E.C. Only the new keyframes change E.C so basically lets say there if I change the default E.C to "cubic" so every new keyframe will be cubic now if i want to change it again to "exponential", the new keyframes will be exponential but the ones i placed earlier will stay cubic

Tbh i don't know if it was necessary to write a whole block but i wanted to be clear so yeah : )

**Comment by andydjames** [comment]
Being able to set defaults for other settings would be great e.g. 
Vector control point types (current default is symmetric - I tend to use asymmetric by default)

**Reply by kavlor** [reply]
Made a similar post about this.It would speed up things instead of having to go to each point and change it.
